### PR TITLE
Add the final UI desin @talihomz

### PR DESCRIPTION
**Pull request by MrObele**
Add the final i design to master

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,351 @@
...
+				<i class="fas fa-th"></i>
+				<i class="fas fa-comment"></i>
+				<i class="fas fa-bell"></i>
+				<img class="user" src="./assets/media/images/user.png" alt="user">
+			</section>
+		</div>
+
+	</header>
+	<main>
+		<content>
```
Comment (talihomz) [inline index.html]: What HTML5 element is this? From my research, the one I found is deprecated https://developer.mozilla.org/en-US/docs/Web/HTML/Element/content

**Review by talihomz — COMMENTED** [review]
@MrObele  @uzorjchibuzor, Good job on this PR!

On the layout, there are a few issues:
1. The navbar has a slight margin top as opposed to how Youtube really is.
2. The navbar height should atleast match Youtube's.
3. The video suggestions on the left are screenshots of the actual thing. Why not just write code for them?
4. The search button is abit larger than the search input box.

Otherwise, this is awesome work for your first HTML project. Don't forget to update the repo's README.

**Comment by uzorjchibuzor** [comment]
@talihomz, Thanks for your comments and observations.

We have made the corrections on the page, also, the readme file has been updated.

Thanks

**Comment by uzorjchibuzor** [comment]
We have been remiss of sending you the links to review guidelines for our completed projects @talihomz , This is the link for YoutubePage review `https://docs.google.com/document/d/1JrKsYPS_t_QOTUm927UIk-e1ZiqZeaTfNufq-P0DxCE/edit`

File: index.html [hunk 2]
```diff
@@ -0,0 +1,450 @@
...
+				<i class="fas fa-bell"></i>
+				<img class="user" src="./assets/media/images/user.png" alt="user">
+			</section>
+		</div>
+
+	</header>
+	<main>
+		<section class="main-video">
+
+			<iframe width="560" height="450" src="https://www.youtube.com/embed/CvJG4sQhzsw" frameborder="0" allow="autoplay; encrypted-media"
```
Comment (heyapricot) [inline index.html]: The frameborder attribute on the iframe element is obsolete. Use CSS instead

File: index.html [hunk 3]
```diff
@@ -0,0 +1,450 @@
...
+					<div class="by-line">
+						<p>TheHappieCat</p>
+						<p class="date-published">Published on 2 Mar 2016</p>
+					</div>
+					<div class="subscribe-button">
+						<button>SUBSCRIBE
+							<span> 412K </span>
+						</button>
+					</div>
+					<div class="clear-float">
```
Comment (heyapricot) [inline index.html:74]: Having one of this between every element of the column feels like a "hack". I don't think generating an extra div for every column element is the best way to go

File: assets/stylesheet/style.css [hunk 4]
```diff
@@ -0,0 +1,360 @@
...
+
+iframe {
+
+  top:0;
+  left:0;
+  width:100%;
+
+}
+
+.topPlay{
```
Comment (heyapricot) [inline assets/stylesheet/style.css]: The convention is to use [kebab-case](http://wiki.c2.com/?KebabCase) for CSS classes. I'm curious why you decided to change the naming convention for this class.

Comment (uzorjchibuzor) [reply assets/stylesheet/style.css]: That was our first project and my partner preferred camelCasing, I convinced him afterwards.

File: index.html (hunk 3 above)
Comment (uzorjchibuzor) [reply index.html:74]: We were using floats and needed to clear it. That was the best we could come up with at the time. What would you advise?

File: index.html (hunk 2 above)
Comment (uzorjchibuzor) [reply index.html]: Frameborder attribute has been removed

File: assets/stylesheet/style.css (hunk 4 above)
Comment (uzorjchibuzor) [reply assets/stylesheet/style.css]: I have made changes to the code, changing the camelCasing to kebab-case on both the CSS and the HTML

File: index.html (hunk 3 above)
Comment (heyapricot) [reply index.html:74]: I mean at this point we can let it go. What I'm trying to say it's that it's not usual to see A LOT of empty <div> in a webpage just to make a design (and in this case, a list) look nice.

File: index.html (hunk 2 above)
Comment (heyapricot) [reply index.html]: 👍

Comment (uzorjchibuzor) [reply index.html]: Thanks

**Pull request merged**